lib/fastlane/plugin/yarn/helper/yarn_helper.rb

```diff
@@ -31,7 +31,8 @@ def check_install
 
         UI.message("Checking yarn install and dependencies")
         begin
-          Action.sh(self.yarn, print_command: true, print_command_output: true)
+          command = [self.yarn, "--version"].compact.join(" ")
```

Comment:
So when I wrote this I threw it together real quick so this check install is kinda doing two things :( Checking to see if yarn is installed at all (which is what you are fixing) but also installs the dependencies in the `package.json` file. So with this change they aren't getting install and yarn commands fail.

Maybe split this method out, check if yarn is actually installed, then install `package.json` dependencies before executing the yarn command.

thoughts?